src/services/category/category.service.ts

```diff
@@ -0,0 +1,44 @@
+import { CreateCategoryDto, UpdateCategoryDto } from '@dtos/category';
+import { Category } from '@entities/category';
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { CategoryRepository } from '@repository/category';
+
+@Injectable()
+export class CategoryService {
+  constructor(
+    @InjectRepository(CategoryRepository)
+    private categoryRepository: CategoryRepository
+  ) {}
+
+  async createCategory(
+    createCategoryDto: CreateCategoryDto
+  ): Promise<Category> {
+    const category = await this.categoryRepository.createCategory(
+      createCategoryDto
+    );
+
+    return category;
```

Comment:
Could this method just return the repository promise and avoid creating a category constant?